src/api/Dockerfile

```
@@ -65,15 +65,17 @@ RUN pip install -e ./core[dev] && \
     done
 
 # Un-privileged user running the application
-USER ${DOCKER_USER:-1000}
+ARG DOCKER_USER
```

You should defined a default value for the [`ARG`](https://docs.docker.com/reference/dockerfile/#arg)
```suggestion
ARG DOCKER_USER=1000
```

---

you're right, thanks!